CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # CHANGELOG
+
+### v1.12.0 (2021-03-15)
+
+- **bug fix** PhoneNumberRegulatoryRequirements now hits `phone_numbers_regulatory_requirements`
+- **resources** Fax support
```

Comment:
I guess this line was not intended to be included

Reply:
we missed fax support in the last release, updating that with this version